debian/rules

```diff
@@ -37,7 +37,7 @@ endif
 endif
 
 override_dh_gencontrol:
-	echo extra:Depends=$(LIBAPT_PKG_DEP) >> debian/ubuntu-advantage-tools.substvars
+	[ -z '$(LIBAPT_PKG_DEP)' ] || echo extra:Depends=$(LIBAPT_PKG_DEP) >> debian/ubuntu-advantage-tools.substvars
```

Comment:
Could also use another `ifeq` here instead of a shell variable check. I had to use single quotes here because the makefile variable has symbols like `(` and `>` which the shell doesn't like.